src/20240914_Multi_OnboardUSDSAndSUSDS/OnboardUSDSAndSUSDS.md

```diff
@@ -62,11 +61,11 @@ Following discussions with Sky teams and Aave DAO service providers, the followi
 - Collateral Usage: Disabled
 - LTV, Liquidation Threshold, and Liquidation Bonus: All reduced to 0%.
 
-#### sUSDS (Aave V3 Main Instance):
+#### USDS (Main Pool):
 
-- Reserve Factor: Increased from 25% to 50%
-- Variable Rate Slope 1: Decreased from 5.5% to 2%
-- Variable Rate Slope 2: Decreased from 75% to 50%
+- Reserve Factor: Decreased from 25% to 10%
+- Base Variable Borrow Rate: Increased from 0% to 0.75%
+- Variable Rate Slope 1: Increased from 5.5% to 6.25%
 - Optimal Usage Ratio: Increased from 90% to 92%
 
 ## References
```

Comment:
Please change the commit hash `efae763a23aefa9fc4506cca5223bcf5a563fea8`, to `main`